scripts/bulk-access-control.md

```diff
@@ -0,0 +1,93 @@
+
+
+## How it Works?
+
+The Bulk Access Control Management script reads a json file containing the details about which resources (items and bitstreams) need to be processed and how to manipulate the access conditions.
+The script then processes the items and bitstreams identified according to the defined mode and specified accessConditions.
+
+## Parameters
+
+The script takes the following parameters:
+
+| Parameter                        | Description                                                                                                                                                                            |
+|----------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-f`, <br>`--file`                | This parameter specifies the json file containing the processing instruction.                                                                                                              |
+| `-h`, <br>`--help`               | This parameter displays help information for the script.                                                                                                                               |
+
+The json file specified with the file parameter has the following structure
+ structured as follow
+```json
+{
+   target: [uuid1, uuid2, ..., uuidN],
+   item: {
+      mode: "replace",
+      accessConditions: [
+          {
+            "name": "openaccess"
+          }
+      ]
+   },
+   bitstream: {
+      constraints: {
+          uuid: [bit-uuid1, bit-uuid2, ..., bit-uuidN],
+      },
+      mode: "add",
+      accessConditions: [
+        {
+         "name": "embargo",
+         "startDate": "2024-06-24T23:59:59.999+0000"
+        },
+        {
+         "name": "network"         
+        }
+      ]
+   }
+}
+```
+* target: is a list of uuid of communities, collections and individual items to process
+* item: the item node contains the processing detail at the item level for the identified resources
+  * mode: can be replace or add. Replace will mean that after processing the items will have only the access conditions that are specified in the accessContitions property. Add will mean that after processing the items will have the specified accessCondition in addition to the ones already defined.
+  * accessConditions: the list of access conditions to apply, with their specific parameters if needed (such as a startDate for an embargo, etc.). Only in replace mode the accessConditions can be empty in this case the access conditions of the item will be reset to what is inherited from their owningCollection. Please note that when accessConditions is set and is not empty not inheritence from the owningCollection occur
```

Comment:
My assumption is this `accessConditions` section uses the same syntax as https://github.com/DSpace/RestContract/blob/main/submissionaccessoptions.md (and is also therefore configurable in the same way)?   If so, we should state that, or add a link from here to that other area of our docs.

UPDATE: I see now that you have this information at the very bottom. That is OK, but then we should add a brief note here to say "For more information on accessConditions see the 'What is an Access Condition?' section below."